.github/workflows/linter.yml

```diff
@@ -0,0 +1,18 @@
+name: eslint Checker
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: Run eslint
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "13.x"
+      - name: Run linter
+        run: |
+          cd client
```

Comment:
Is it possible to cache the node_modules?

Reply:
I'll make it in another PR. Should've set request changes! oops